build/configs/esp32_DevKit/README.md

```diff
@@ -0,0 +1,110 @@
+# esp32_DevKit
+
+Espressif Development Kit for ESP32 chipset.
+
+## Contents
+
+> [Information](#information)  
+> [Environment Set-up](#environment-set-up)  
+ 
+## Information
+
+will be updated
+
+## Environment Set-up
+
+### 1. Getting the toolchain
+Get the build in binaries and libraries
+
+64-bit Linux：
+
+[xtensa-esp32-elf-linux64-1.22.0-80-g6c4433a-5.2.0.tar.gz](https://dl.espressif.com/dl/xtensa-esp32-elf-linux64-1.22.0-80-g6c4433a-5.2.0.tar.gz)
+
+32-bit Linux：
+
+[xtensa-esp32-elf-linux32-1.22.0-80-g6c4433a-5.2.0.tar.gz](https://dl.espressif.com/dl/xtensa-esp32-elf-linux32-1.22.0-80-g6c4433a-5.2.0.tar.gz)
+
+```bash
+tar xvjf xtensa-esp32-elf-[linux*]-1.22.0-80-g6c4433a-5.2.0.tar.gz
+export PATH=<Your Toolchain PATH>:$PATH
+```
+
+### 2. Getting TizenRT source code
+
+```bash
+git clone https://github.com/Samsung/TizenRT.git
+cd TizenRT
+TIZENRT_BASEDIR="$PWD"
+```
+**Note**: To contribute in this community, you need to clone your forked private repository instead.  
+          Github guides this by [working-with-forks](https://help.github.com/articles/working-with-forks).
+
+### 3. How to Build
+
+Configure the build from $TIZENRT_BASEDIR/os/tools directory
+
+```bash
+cd os
+./tools/configure.sh <board>/<configuration_set>
+```
+
+This command retrieves the specific pre-configured file named defconfig and Make.defs according to \<board\>/\<configuration_set\>.  
+You can see collection of all configuration files at *$TIZENRT_BASEDIR/build/configs*.  
+To check all pre-defined configurations, type as follows:
+
+```bash
+./configure.sh --help
+```
+
+After configuring above, configuration can be modified through *make menuconfig* from *$TIZENRT_BASEDIR/os*.
+Howerver, menuconfig isn't ready for Xtensa architecture esp32 board. Please do not use it for esp32 board.
```

Comment:
I think you have added menuconfig feature already.

Reply:
fixed, thanks.